src/app/zap-templates/zcl/data-model/chip/electrical-power-measurement-cluster.xml

```diff
@@ -52,9 +52,18 @@ limitations under the License.
 
     <!--Attributes-->
     <attribute code="0x0000" side="server" define="POWER_MODE" type="PowerModeEnum" min="0x00" max="0x02">PowerMode</attribute>
-    <attribute code="0x0001" side="server" define="NUMBER_OF_MEASUREMENT_TYPES" type="int8u" min="1">NumberOfMeasurementTypes</attribute>
-    <attribute code="0x0002" side="server" define="ACCURACY" type="array" entryType="MeasurementAccuracyStruct" minLength="1">Accuracy</attribute>
-    <attribute code="0x0003" side="server" define="RANGES" type="array" optional="true" entryType="MeasurementRangeStruct" minLength="0">Ranges</attribute>
+    <attribute code="0x0001" side="server" define="NUMBER_OF_MEASUREMENT_TYPES" type="int8u" min="1">
+        <description>NumberOfMeasurementTypes</description>
+        <quality persistence= "fixed" />
```

Comment:
please fix extra spacing (should be `persistence="fixed"` without the extra space after =)